scipy-1.0/paper.tex

```diff
@@ -735,52 +735,52 @@ \subsection*{Submodule organization}
 
 \subsection*{Language choices}
 
-Python, Cython, Fortran, C and C++ are the programming languages used to
-implement scientific algorithms in the SciPy library. An analysis of our code
-base using the \texttt{linguist} library\cite{linguistref} provides a 
-detailed breakdown as \% composition by programming language in 
-SciPy (Figure~\ref{fig:linguist}).
+Figure~\ref{fig:linguist} provides a breakdown of the programming languages used
+in SciPy. While it is not surprising that Python is predominant, the usage distribution 
+of the other languages warrants discussion. 
 
 \begin{figure}[H]
-    \centering
-
-    \begin{tikzpicture}[font=\sffamily]
-    \begin{axis}[height=6cm, width=10cm,
-    ybar, ymax=60, ymin=0,bar width=1cm,
-    ylabel={\% of Scipy code base},
-    ytick={0,20,...,60},
-    axis lines*=left,
-    axis y line shift=0pt,
-    enlarge x limits=0.15,
-    symbolic x coords={py,f,c,cy,cpp},
-    xticklabels={Python,FORTRAN,C,Cython,C++},
-    ticklabel style={align=center,font=\sffamily\small,
-    /pgf/number format/assume math mode=true},
-    xtick=data,
-    nodes near coords,
-    nodes near coords style={/pgf/number format/assume math mode=true}
-    ]
-    \addplot coordinates {(py,49.5) (f,25.6) (c,19.5) (cy, 3.0) (cpp,2.3)};
-    \end{axis}
-    \end{tikzpicture}
-    \caption{The breakdown of programming languages used in the
-             SciPy library determined using the linguist library.
-    	 Small ($<0.5 \%$) amounts of TeX, Matlab, Shell,
-    	 and Makefile are excluded for clarity and mostly
-    	 provide supporting roles in tests, building, and
-    	 documentation.}
-    \label{fig:linguist}
+\centering
+\includegraphics[width=0.5\textwidth]{static/scipy_languages}
+\caption{Composition of SciPy: lines of code in each programming language as determined by the \texttt{linguist} package.
+Small ($<0.5 \%$) amounts of TeX, Matlab, Shell,
+and Makefile (not visible) provide supporting roles in tests,
+building, and documentation.}
+\label{fig:linguist}
 \end{figure}
 
-For implementing new functionality, we have a clear order of language
-preference.  First Python, if performance is not an issue. If it is, then in
-order of preference: Cython, C, C++, Fortran. The main motivation for this is
-maintainability: Cython has the highest abstraction level and most Python
-developers will understand it. C is also widely known, and easier to deal with
-for the core development team than C++ and especially Fortran.
-
-While it is not surprising that Python is heavily used in SciPy, the usage
```

Comment:
Perhaps this is meant to address the difference between language preference order and actual composition, but I did not see that in previous reads, so I think it needs to be more explicit.